Kubernetes/windows/start-kubeproxy.ps1

```diff
@@ -1,9 +1,10 @@
 Param(
```

Comment:
So, there are going to be 2 start-kubeproxy.ps1 files. When should we use either of them?

Plus, there are also ``start-kubelet.ps1``, ``start.ps1``, ``stop.ps1`` in the windows folder. Are we not going to remove them?

Reply:
The scripts in the windows folder are for wincni directly without flannel delegation. Right now, they are not in the mainline docs so they shouldn't be used unless there is a specific requirement.